core/ledger/pvtdatastorage/store.go

```diff
@@ -521,6 +550,35 @@ func (s *Store) getMissingData(group []byte, maxBlock int) (ledger.MissingPvtDat
 	return missingPvtDataInfo, nil
 }
 
+// FetchBootKVHashes returns the KVHashes from the data that was loaded from a snapshot at the time of
+// bootstrapping. This funciton returns an error if the supplied blkNum is greater than the last block
+// number in the booting snapshot
+func (s *Store) FetchBootKVHashes(blkNum, txNum uint64, ns, coll string) (map[string][]byte, error) {
```

Comment:
Currently, this function is not used in the production code. As it is an exported method, is the goal to make kvledger use this method to check on the reconciled data to identify matching/mismatching hashes?